paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/activity/PaymentSheetPlaygroundActivity.kt

```diff
@@ -131,6 +140,69 @@ internal class PaymentSheetPlaygroundActivity : AppCompatActivity() {
         disableViews()
     }
 
+    override fun onResume() {
+        super.onResume()
+        val sharedPreferences = getSharedPreferences(sharedPreferencesName, MODE_PRIVATE)
+
+        val customer = sharedPreferences.getString("customer", CheckoutCustomer.Guest.value)
+        val googlePay = sharedPreferences.getBoolean("googlePayConfig", true)
+        val currency = sharedPreferences.getString("currency", CheckoutCurrency.USD.value)
+        val mode = sharedPreferences.getString("mode", CheckoutMode.Payment.value)
+        val setShippingAddress = sharedPreferences.getBoolean("setShippingAddress", true)
+        val setAutomaticPaymentMethods = sharedPreferences.getBoolean("setAutomaticPaymentMethods", true)
+
+        setToggles(customer, googlePay, currency, mode, setShippingAddress, setAutomaticPaymentMethods)
```

Comment:
Does this mean there are two places defaults are defined - here and on line 91? If yes, is there a way we can have a single source of truth?

Reply:
Similarly, we have two places that define the keys (e.g. "googlePayConfig") - ideas for having one source of truth?

Reply:
^ +1 to this. Maybe we could create a toggle model class or something? That way we have mapping for toggle -> default value.